ams/server/pom.xml

```diff
@@ -377,6 +377,14 @@
             <classifier>tests</classifier>
             <scope>test</scope>
         </dependency>
+
+        <!-- https://mvnrepository.com/artifact/com.mysql/mysql-connector-j -->
+        <dependency>
```

Comment:
You should not add this dependency to pom.
BTW, you can configure it in the dependencies of your IDEA project/module.